lesson_13.md

```diff
@@ -10,7 +10,7 @@ Students will be able to...
 
 ## Materials/Preparation
 
--   [Do Now 1.3: Drawing a Triangle](do_now_13.md)  
+-   [Do Now 1.3: Drawing a Triangle](do_now_13.md) You can ask students to gives instructions in plain language, pseudo code or with images
```

Comment:
"... students to give instructions ...". Also "plain language, pseudo code, or images".